softvis3d-frontend/src/classes/status/LoadAction.ts

```diff
@@ -14,16 +14,32 @@ export default class LoadAction extends StatusAction {
         this._current = current;
     }
 
-    public get limit(): number {
+    public get limit(): number | undefined {
```

Comment:
The thing for me here is that it can be undefined in the "real world". So our code has to handle this and it has to be checked in the code. 

If i would set max and current to 0 i am not longer able to know if the progress bar should be visible or not - or the code would check for 0 as value. With undefined, this is clear - at least for me ;-).

I cleaned up a bit, so maybe this one is more clear now.